src/doc/dev_manual/ContributingHostProfiles.rst

```diff
@@ -0,0 +1,16 @@
+Contributing Host Profiles
+==========================
+
+Host profiles live in /src/resources/hosts/<compute center name>.
+
+There are a couple of helper file and scripts in the 'hosts' directory that 
+allow the centers and profiles to be added during installation of VisIt, 
+or imported later via the gui.
+
+* If you add or remove directories from 'hosts', please edit networks.dat accordingly.
+
+* If you add or remove directories or files, please regenerate networks.json by running dump_dir_to_networks_json.py
+
+* Note that if run on Windows, the generated networks.json file will have dos-style line endings. Convert it to Unix style before committing.
```

Comment:
I believe our style is to put each sentence on its own line, so put the second sentence on a new line.

Reply:
That is our style, but I don't know hot to create a new line and still have the text be part of the same bullet.

Reply:
Would it make sense if that last sentence was not part of the bulleted list?

Reply:
> I believe our style is to put each sentence on its own line, so put the second sentence on a new line.

Yeah, this is something we discussed a while back and I added [this info](https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/gui_manual/Contributing/index.html#about-line-length) to our contributing guidelines.

Reply:
I hadn't realized that splitting up bullets across multiple lines didn't work. I think it is ok then. Sorry about the noise. I learned something (or perhaps relearned).

Reply:
I didn't realize it either till I made the change and sphinx spit out a warning.

Reply:
To @biagas question, you can split [bulleted text across multiple lines](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#bullet-lists) in restructured text